### PR TITLE
unflag es6 Node >= 13.2.0

### PR DESCRIPTION
**Pull request by TJKoury**
A few changes required for using with Node >= 13.2.0.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @TJKoury!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by mramato** [comment]
Thanks @TJKoury, I assume this fixes #8452?

File: .gulp.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "flags": {
+    "gulpfile": "gulpfile.cjs"
+  }
+}
```
Comment (mramato) [inline .gulp.json]: What looks for this file?  Is it a gulp thing?

Comment (TJKoury) [reply .gulp.json]: Yes, they added it a few versions back.

**Comment by TJKoury** [comment]
This issue is causing the problem: [requizzle with Node > 10].(https://github.com/hegemonic/requizzle/issues/6)

Updating JSDoc to @latest to include latest version of requizzle throws multiple errors.  

Renaming or converting [/Tools/cesium_template/publish.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Tools/jsdoc/cesium_template/publish.js) does not play nice with current version of JSDoc.

**Comment by TJKoury** [comment]
@mramato It does appear to be the same issue as [#8452](https://github.com/AnalyticalGraphicsInc/cesium/issues/8452).

**Comment by mramato** [comment]
I would love to know the motivation behind this massive breaking change in Node 13.

For now I think we may just have to wait for everyone else to play catchup until all of our own dependencies work properly.

**Comment by TJKoury** [comment]
@mramato That seems to be the prudent approach.  I will monitor and update as appropriate.

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @TJKoury! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by mramato** [comment]
Apparently 12.16 (possible 12.10 and later) also have this issue now.

**Comment by TJKoury** [comment]
Just read the other thread.

**Comment by TJKoury** [comment]
Going to close this and reopen.

**Pull request closed without merging**